engine/hooks/tasks.go

```diff
@@ -105,6 +107,32 @@ func (s *Service) synchronizeTasks() error {
 		s.Dao.SaveTask(t)
 	}
 
+	// Start listening to gerrit event stream
+	vcsConfig, err := s.Client.VCSConfiguration()
+	if err != nil {
+		return sdk.WrapError(err, "unable to get vcs configuration")
+	}
+
+	for k, v := range vcsConfig {
+		if v.Type == "gerrit" && v.Username != "" && v.Password != "" && v.SSHPort != 0 {
+			// open event stream
+			vcsName := k
+
+			// Create channel to store gerrit event
+			gerritEventChan := make(chan GerritEvent, 20)
+
+			// Listen to gerrit event stream
+			sdk.GoRoutine(ctx, "gerrit.EventStream."+vcsName, func(ctx context.Context) {
```

Comment:
how do we stop this if we stop or remove the hook ?

Reply:
we never stop. If you fill gerrit configuration, you listen gerrit.